### PR TITLE
Updating galaxy and README.md

### PR DESCRIPTION
**Pull request by ABHISHEK-SINHA10**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: docs/README.md [hunk 1]
```diff
@@ -10,135 +10,102 @@ You may obtain a copy of the License at
...
+| [dellemc_system_lockdown_mode](modules/dellemc_system_lockdown_mode.rst)                             | ✓      | ✓      |
+| [idrac_attributes](modules/idrac_attributes.rst)                                                     | ✓      | ✓      |
+| [idrac_bios](modules/idrac_bios.rst)                                                                 | ✓      | ✓      |
+| [idrac_boot](modules/idrac_boot.rst)                                                                 | ✓      | ✓      |
+| [idrac_certificates](modules/idrac_certificates.rst)                                                 | ✕      | ✓      |
+| [idrac_diagnostics](modules/idrac_diagnostics.rst)                                                   | ✕      | ✓      |
+| [idrac_firmware](modules/idrac_firmware.rst)                                                         | ✓      | ✓      |
+| [idrac_firmware_info](modules/idrac_firmware_info.rst)                                               | ✓      | ✓      |
+| [idrac_lifecycle_controller_jobs](modules/idrac_lifecycle_controller_jobs.rst)                       | ✕      | ✓      |
```
Comment (rajshekarp87) [inline docs/README.md]:
```suggestion
| [idrac_lifecycle_controller_jobs](modules/idrac_lifecycle_controller_jobs.rst)                       | ✓      | ✓      |
```

File: docs/README.md [hunk 2]
```diff
@@ -10,135 +10,102 @@ You may obtain a copy of the License at
...
+| [idrac_storage_volume](modules/idrac_storage_volume.rst)                                             | ✓      | ✓      |
+| [idrac_syslog](modules/idrac_syslog.rst)                                                             | ✓      | ✓      |
+| [idrac_system_info](modules/idrac_system_info.rst)                                                   | ✓      | ✓      |
+| [idrac_timezone_ntp](modules/idrac_timezone_ntp.rst)                                                 | ✓      | ✓      |
+| [idrac_user](modules/idrac_user.rst)                                                                 | ✓      | ✓      |
+| [idrac_user_info](modules/idrac_user_info.rst)                                                       | ✓      | ✓      |
+| [idrac_virtual_media](modules/idrac_virtual_media.rst)                                               | ✓      | ✓      |
+| [redfish_event_subscription](modules/redfish_event_subscription.rst)                                 | ✕      | ✓      |
+| [redfish_firmware_rollback](modules/redfish_firmware_rollback.rst)                                   | ✕      | ✓      |
```
Comment (rajshekarp87) [inline docs/README.md]:
```suggestion
| [redfish_firmware](modules/redfish_firmware.rst)                                   | ✕      | ✓      |
```

File: docs/README.md [hunk 3]
```diff
@@ -10,135 +10,102 @@ You may obtain a copy of the License at
...
+| [redfish_firmware](modules/redfish_firmware.rst)                                                     | ✓      | ✓      |
```
Comment (rajshekarp87) [inline docs/README.md]:
```suggestion
| [redfish_firmware_rollback](modules/redfish_firmware_rollback.rst)                                                     | ✓      | ✓      |
```

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -1772,3 +1772,31 @@ releases:
           name: idrac_user
           namespace: null
     release_date: '2024-02-29'
+  9.1.0:
+    changes:
+      release_summary: '- idrac_storage_volume - Configures the RAID configuration attributes.'
```
Comment (rajshekarp87) [inline changelogs/changelog.yaml]:
```suggestion
      release_summary: '- ``redfish_storage_volume`` is enhanced to support iDRAC8.
      
      - ``dellemc_idrac_storage_module`` is deprecated and replaced with ``idrac_storage_volume``.'
```

**Review by rajshekarp87 — DISMISSED** [review]

**Review by felixs88 — DISMISSED** [review]

**Review by rajshekarp87 — DISMISSED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Review by felixs88 — APPROVED** [review]

**Pull request merged**